### DISCUSSION TITLE
Airplay 2 synchronization off

### DISCUSSION THREAD

**Discussion by DeanSCND** [mikebrady/shairport-sync#1392] [Q&A]
I have 4 instances of sp development branch running ap2. I have 2 positioned in the same room. Airplay connection is continually solid but I'm noticing about a quarter second delay on playback. I have tested both on raspian and dietpi with same delay. Wired and wireless combinations produce same result. 

Any advice on debugging or configuration changes to get them lined up?

**Comment by charlesomer** [comment]
Is this a delay between the two in the same room? If so, are they connected to two different types of speakers as different equipment can have different amounts of delay after Shairport Sync.

`audio_backend_latency_offset_in_seconds = 0.0;` can be used in the config file to correct for this 😊 
https://github.com/mikebrady/shairport-sync/blob/master/scripts/shairport-sync.conf

**Reply by DeanSCND** [reply]
Thanks for the reply and sorry for delay. Christmas got in the way. The speakers are different ... the streamers are identical. I'm going to give the delay suggestion a go tomorrow. Thanks again for the suggestion. Update after.
